src/TuDa.CIMS.Web/Components/Dashboard/Dialogs/AssetItemCreateForm.razor.cs

```
@@ -179,29 +197,9 @@ public async Task SaveChanges()
                 RoomId = RoomId,
             },
 
-            _ => null,
+            _ => null, // Should never happen!
```

🤔 Maybe throw an exception here or return an Error